common/app/routes/challenges/components/step/Step.jsx

```diff
@@ -1,9 +1,11 @@
 import React, { PropTypes } from 'react';
+import ReactDOM from 'react-dom';
 import classnames from 'classnames';
 import { connect } from 'react-redux';
 import { createSelector } from 'reselect';
 import PureComponent from 'react-pure-render/component';
 import LightBox from 'react-images';
+import MouseTrap from 'mousetrap';
```

Comment:
Mousetrap capture is done in https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/client/sagas/mouse-trap-saga.js

You'll need to capture them there and then dispatch appropriate actions